### PR TITLE
Java doc

### PR DESCRIPTION
**Pull request by Geal**

### CODE REVIEW

File: content/docs/Usage/java.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    Biscuit token = Biscuit.from_bytes(serializedToken, root.public_key());
+    Block block = token.create_block().add_check("check if operation(\"read\")");
+    return token.attenuate(block);
+}
+```
+
+## Seal a token
+
+```java
+Either<Error, byte[]> sealed_token = token.seal();
```
Comment (KannarFr) [inline content/docs/Usage/java.md:77]: It has been removed, isn't useless ATM?

File: content/docs/Usage/java.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+## Reject revoked tokens
+
+The `revocation_identifiers` method returns the list of revocation identifiers as byte arrays.
+
+```java
+List<byte[]> revocation_ids = token.revocation_identifiers();
```
Comment (KannarFr) [inline content/docs/Usage/java.md:85]: No it returns `List<RevocationIdentifier>`

**Review by KannarFr — CHANGES_REQUESTED** [review]
There is no mention of `UnverifiedBiscuit`, its usage, and its `verify(publicKey)` method (which returns Biscuit)?

File: content/docs/Usage/java.md (hunk 1 above)
Comment (Geal) [reply content/docs/Usage/java.md:77]: why was it removed?

**Comment by Geal** [comment]
> There is no mention of UnverifiedBiscuit, its usage, and its verify(publicKey) method (which returns Biscuit)?
to be fair the `UnverifiedBiscuit` class did not exist when I wrote those examples :D

**Pull request merged**